cns/ipampoolmonitor/ipampoolmonitor_test.go

```diff
@@ -395,3 +434,51 @@ func TestPoolSizeDecreaseToReallyLow(t *testing.T) {
 		t.Fatalf("Expected IP's not in use to be 0 after reconcile, expected %v, actual %v", (initialIPConfigCount - batchSize), len(poolmonitor.cachedNNC.Spec.IPsNotInUse))
 	}
 }
+
+func TestDecreaseAfterNodeLimitReached(t *testing.T) {
```

Comment:
Can we also add a test for increase where maxPodIPCount is not a multiple of batchSize?

Reply:
1. We already have that, check `TestPoolIncreasePastNodeLimit`, BatchSize = 16, MaxPodIpCount = 30
2. I think you mean where BatchSize > MaxPodIpCount? If so I'll add it in

Reply:
Added